rest/handler.go

```diff
@@ -638,6 +638,23 @@ func (h *handler) readSanitizeJSON(val interface{}) error {
 	return err
 }
 
+// readJavascript reads a javascript function from a request body.
+func (h *handler) readJavascript() (string, error) {
+	// Performs the Content-Type validation and Content-Encoding check.
+	input, err := processContentEncoding(h.rq.Header, h.requestBody, "text/javascript")
```

Comment:
I thought text/javascript was deprecated in favour of application/javascript.  
https://www.iana.org/assignments/media-types/media-types.xhtml#application
https://www.iana.org/assignments/media-types/media-types.xhtml#text

I might be missing a subtlety there, though - let me know.  Could also consider accepting both, if that's appropriate.